scripts/actions/src/add-default-roles.ts

```diff
@@ -32,7 +32,7 @@ exports.onExecutePostLogin = async (
       scope: 'read:users update:users read:roles',
     })
 
-    const params = { id: event.user!.user_id }
+    const params = { id: event.user.user_id }
```

Comment:
According to the [Auth0 docs](https://auth0.com/docs/customize/actions/flows-and-triggers/login-flow/event-object), this is not optional, so I think this is a bug in the type library